numpy/testing/_private/utils.py

```diff
@@ -1639,98 +1639,6 @@ def integer_repr(x):
         raise ValueError("Unsupported dtype %s" % x.dtype)
 
 
-# The following two classes are copied from python 2.6 warnings module (context
-# manager)
-class WarningMessage(object):
```

Comment:
Need to import this from `warnings`.

Reply:
Did that, waiting on tests.

Reply:
Thanks, guess my search for uses missed that one